app/components/post_draft/draft_input/draft_input.js

```diff
@@ -329,12 +329,13 @@ export default class DraftInput extends PureComponent {
         const notificationsToChannel = enableConfirmNotificationsToChannel && useChannelMentions;
         const notificationsToGroups = enableConfirmNotificationsToChannel && useGroupMentions;
         const toAllOrChannel = DraftUtils.textContainsAtAllAtChannel(value);
-        const groupMentions = (!toAllOrChannel && notificationsToGroups) ? DraftUtils.groupsMentionedInText(groupsWithAllowReference, value) : [];
+        const toHere = DraftUtils.textContainsAtHere(value);
+        const groupMentions = (!toAllOrChannel && !toHere && notificationsToGroups) ? DraftUtils.groupsMentionedInText(groupsWithAllowReference, value) : [];
```

Comment:
what if I want to mention `@here` and `@group` ? which is different than `@channel|all` right ?

Reply:
Going by the pre-existing logic I am assuming `@all`, `@channel` and `@here` take precedence of the group mention if they trigger the alert.

Reply:
to me this is the difference, I may want to alert those online by mentioning `@here` and also explicitly the `@group`, when  using an `@channel` or `@all` the notification is sent to the entire channel including the group, that is why I ask

Reply:
Understood. @esethna thoughts?

Reply:
Missed this notification. In my mind this is just about adding a warning dialog that you **might be** alerting a lot of people. If there's an @here it means there's a potential to alert a large number of users, so we show a warning.

Reply:
So if you use `@here` we will never look into the group mentions, if that is OK then I'll approve, but still seems wrong to me, even with @esethna explanation

Reply:
Sorry I guess I'm not understanding the case you're talking about, can you help with more details and an example? @enahum

Reply:
In my mind if @here is used (whether or not there is also an @group in the message) we are warning the user that they may be notifying a lot of users